### PR TITLE
Upgrade Gorilla Handlers to `1.5.1`.

### PR DESCRIPTION
**Pull request by olegburov**
This PR upgrades [Gorilla Handlers](https://github.com/gorilla/handlers) from [1.1.0](https://github.com/gorilla/handlers/commit/60c7bfde3e33c201519a200a4507a158cc03a17b) to [1.5.1](https://github.com/gorilla/handlers/releases/tag/v1.5.1)

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -41,6 +41,8 @@ github.com/docker/go-metrics v0.0.1 h1:AgB/0SvBxihN0X8OR4SjsblXkbMvalQ8cjmtKQ2rQ
 github.com/docker/go-metrics v0.0.1/go.mod h1:cG1hvH2utMXtqgqqYE9plW6lDxS3/5ayHzueweSI3Vw=
 github.com/docker/libtrust v0.0.0-20150114040149-fa567046d9b1 h1:ZClxb8laGDf5arXfYcAtECDFgAgHklGI8CxgjHnXKJ4=
 github.com/docker/libtrust v0.0.0-20150114040149-fa567046d9b1/go.mod h1:cyGadeNEkKy96OOhEzfZl+yxihPEzKnqJwvfuSUqbZE=
+github.com/felixge/httpsnoop v1.0.1 h1:lvB5Jl89CsZtGIWuTcDM1E/vkVs49/Ml7JJe07l8SPQ=
+github.com/felixge/httpsnoop v1.0.1/go.mod h1:m8KPJKqk1gH5J9DgRY2ASl2lWCfGKXixSwevea8zH2U=
```
Comment (thaJeztah) [inline go.sum:45]: new dependency; looks like it's MIT, so should be ok; https://github.com/felixge/httpsnoop/

**Comment by milosgajdos** [comment]
Can we bump this to `v1.5.1` @olegburov? Looks like `v1.5.1` fixes a compression issue introduced in `v1.5.0`. See [here](https://github.com/gorilla/handlers/releases/tag/v1.5.1) and they strongly recommend bumping the patch.

**Comment by olegburov** [comment]
@milosgajdos done.

**Review by arkodg — APPROVED** [review]

**Review by milosgajdos — APPROVED** [review]
LGTM

**Review by joaodrp — APPROVED** [review]

**Pull request merged**